vendor/engines/projects/spec/projects_spec_helper.rb

```diff
@@ -0,0 +1,3 @@
+RSpec.configure do |config|
+  config.before(:all) { Projects::Engine.enable! }
```

Comment:
Oh, here's where it gets enabled in specs. Since we're using the existence of the gem in the gemfile we should always `enable!` it. The big reason we used `enable!`/`disable!` in split accounts was because we were using a separate feature flag.